src/wallet/wallet.h

```diff
@@ -942,8 +740,8 @@ class CWallet final : public FillableSigningProvider, private interfaces::Chain:
         encrypted_batch = nullptr;
```

Comment:
I don't think there's any reason for CWallet::encrypted_batch to exist as a member variable anymore. It could just be a local in CWallet::EncryptWallet

Reply:
Done